src/app.ts

```diff
@@ -72,22 +73,25 @@ class App {
         return res.send(swaggerUi.generateHTML(swaggerJSONDoc, swagger_options))
       }
     )
-    this.express.use(Authentication.handleError)
-    this.express.use(Authentication.accessControl)
+    this.express.use(auth.handleError)
+    this.express.use(async (req, res, next) => await auth.accessControl(req, res, next))
   }
 
   private routes() {
     const app = this.express
+    const jsonMiddleware = express.json({ limit: '50mb' })
```

Comment:
This is not correct.

parsing middlewares are used according to the content type of request body. Need not separate them out and specify in each api